shuup/core/models/_product_media.py

```diff
@@ -149,7 +146,7 @@ def get_thumbnail(self, **kwargs):
 
         try:
             return thumbnailer.get_thumbnail(thumbnail_options=kwargs)
-        except InvalidImageFormatError:
+        except (InvalidImageFormatError, OSError):
```

Comment:
Reconsider catching this `OSError` I think this is only eating exceptions in case the images can't be thumbnailed and in theory we don't want those at the first place.